app/controllers/claims/support/schools/mentors_controller.rb

```diff
@@ -3,11 +3,26 @@ class Claims::Support::Schools::MentorsController < Claims::Support::Application
 
   before_action :set_mentor, only: %i[show remove destroy]
   before_action :authorize_mentor
+  before_action :mentor_form, only: %i[new]
 
   def index
     @pagy, @mentors = pagy(@school.mentors.order(:first_name, :last_name))
   end
 
+  def new; end
+
+  def create
+    mentor_form.save!
+
+    redirect_to claims_support_school_mentors_path(@school), flash: { success: t(".mentor_added") }
```

Comment:
Previously, we've started using `.success`.

```suggestion
    redirect_to claims_support_school_mentors_path(@school), flash: { success: t(".success") }
```